changelog.d/pa-3283.fixed

```diff
@@ -0,0 +1,23 @@
+taint-mode: Removed a hack that made `lval = new ...` assignments to not clean
+the `lval` despite the RHS was not tainted. This caused FPs in double-free rules.
+For example, given this source:
+
+```yaml
+pattern-sources:
```

Comment:
Would it be helpful to make this example rule complete by adding a pattern-sink as well? so users can easily copy and paste it for testing.